local/owr_local_media_source.c

```diff
@@ -631,6 +683,7 @@ static GstElement *owr_local_media_source_request_source(OwrMediaSource *media_s
                     /* Accepting any format didn't work, we're going to hope that scaling fixes it */
                     CREATE_ELEMENT(source_process, "videoscale", "video-source-scale");
                     gst_bin_add(GST_BIN(source_pipeline), source_process);
+                    /* should be gst_object_unref(source_process); ? */
```

Comment:
You don't need the unref -- adding to a bin "sinks" the floating reference you get when you create the element.